staging/operator-lifecycle-manager/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/onsi/gomega v1.10.2
 	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/operator-framework/api v0.0.0-00010101000000-000000000000
+	github.com/operator-framework/api v0.7.1
```

Comment:
Hmm, this should still be using a fake pseudo tag as we replace this reference with the local filesystem package(s).